src/main/java/de/unistuttgart/moorhuhnbackend/data/ConfigurationDTO.java

```diff
@@ -0,0 +1,30 @@
+package de.unistuttgart.moorhuhnbackend.data;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.Nullable;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
```suggestion
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
```
parameters should be private